Documentation/ApiOverview/Environment/Index.rst

```diff
@@ -0,0 +1,90 @@
+.. include:: ../../Includes.txt
+
+.. _Environment:
+
+===========
+Environment
+===========
+
+Since version 9.x the TYPO3 CMS core includes an environment class.
+This class contains all environment specific information, e.g. paths within the
+filesystem. This implementation replaces previously used global variables and
+constants like :php:`PATH_site`.
+
+The fully qualified class name is :php:`\TYPO3\CMS\Core\Core\Environment`. The
+class provides static methods to access the necessary information.
+
+Also for testing, the :php:`initialize()` method can be called to adjust the
+information.
+
+.. _Environment-project-path:
+
+project path
+------------
+
+The environment provides the path to the folder containing the
+:file:`composer.json`, or public web folder for non composer projects.
+
+The path is available through :php:`getProjectPath()` method.
```

Comment:
I am missing a `the` everywhere. Also I'd always write `Environment::getFoo()` to make the usage obvious and even allow for copy and paste.

Reply:
The only thing I didn't got is about `the`, what do you mean with

> I am missing a the everywhere.

Reply:
It should be _The path is available through **the** `getProjectPath()` method._